### ISSUE TITLE
Chrome can misrecognise the username field at registration as an email address and autofill, causing confusion

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#741]
I've heard one report of Chrome autofilling the username field at registration with an email address, causing confusion.

**Comment by richvdh** [comment]
Should probably apply matrix-org/matrix-react-sdk#520 to the registration form

**Comment by kittykat** [comment]
Closing as it should have been fixes by the PR above and I cannot reproduce the issue.
